python/nano/src/bigdl/nano/pytorch/inference/optimizer.py

```diff
@@ -772,17 +775,57 @@ def trace(model: nn.Module,
         invalidInputError(False, "Accelerator {} is invalid.".format(accelerator))
 
     @staticmethod
-    def get_context(model: nn.Module):
+    def get_context(model: nn.Module, *args):
```

Comment:
should we just call it `*models`? No strong idea about this.

Reply:
I think *models might be a good chioce. Do you think we should use 
`get_context(model: nn.Module, *models)`
or just
`get_context(*models)`
as input paramater?